observatorium/overlays/moc/smaug/loki/deployments/observatorium-loki-query-frontend.yaml

```diff
@@ -0,0 +1,124 @@
+kind: Deployment
```

Comment:
seems like this already exists on base path
https://github.com/operate-first/apps/blob/master/observatorium/base/instance/loki/query-frontend/deployment.yaml